react/Icons/Forbidden.tsx

```diff
@@ -1,6 +1,6 @@
-import React, { FunctionComponent } from 'react'
+import React, { FC } from 'react'
```

Comment:
😞

Reply:
It's easier to type................................................ but I don't have another strong argument in favour of `FC` instead of `FunctionComponent`. I usually dislike the way our components signatures end being formatted with a lot of line breaks.

Reply:
😞